ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -114,6 +139,18 @@ const getTransactionGroupRecipientAddressFilter = (
   };
 };
 
+const getTransactionGroupRecipientAddressFilterAllChain = (
+  recipientAddress,
+) => {
+  return ({ initialTransaction: { txParams } }) => {
+    return (
+      isEqualCaseInsensitive(txParams?.to, recipientAddress) ||
+      (txParams?.to === SWAPS_CHAINID_CONTRACT_ADDRESS_MAP &&
+        txParams.data.match(recipientAddress.slice(2)))
```

Comment:
What are we slicing here? Is it like an 0x address and we want to chop off the `0x`?

Reply:
yes , this code was introduced 4 years ago, the only thing i'm changing is making it multichain

Reply:
4 years ago wow lol.
Fair enough.